<launch>
    <node name="rbe3002_planning_node" pkg="rbe3002" type="planning_node.py" output="screen"/>
    <node name="rbe3002_move_node" pkg="rbe3002" type="move_node.py" output="screen"/>
</launch>
